lib/portage/debug.py

```diff
@@ -118,7 +118,7 @@ def __init__(self, prefix):
         self.previous = None
         self.previous_trimmed = None
 
-    def trim(self, s):
+    def trim(self, s: str) -> str:
```

Comment:
Ooh. I wonder if this is actually useful for stuff like https://github.com/gentoo/portage/pull/798#discussion_r841142289. But debug.py is a bit of a scary name for it. So just something I'll keep in mind (where our utilities functions are and if we should regroup them at some point).